index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@200;400;500;700&display=swap" rel="stylesheet">
+</head>
+<body class="main__container">
+    <!-- Creating the navigation menu bar -->
+  <header class="container_menu">
+    <nav class="container--nav">
+        <ul class="navegation--list">
+            <li class="navigation--option"><a>Portfolio</a></li>
+            <li class="navigation--option"><a>About</a></li>
+            <li class="navigation--option"><a>Contact</a></li>
+        </ul>
+        <div class="menu--image">
+            <img src="images\menuUnion.png" alt="Menu button" id="menu--button"/>
+        </div>
+    </nav>
+  </header>
```

Comment:
- [ Required ] I think it would be better if you made your navigation menu appear exactly as it is in the Figma design as requested by a previous reviewer, to achieve this I would like to suggest the following changes to your CSS
  
   - Remove the background image from the `body` tag and add it to the `main` container, then adjust the `CSS` 
   as shown below.

```CSS
/* remove the fixed positioning from the header */
.container_menu {
  padding: 24px 0; 
  width: 100%;
  display: flex;
  flex-direction: row-reverse;
}

/* Remove the fixed height and width from the nav */
.container--nav {
  width: 100%;
}

.main__containerinfo {
  padding-top: 100px;
  background-image: url('images/mobile_background.png');
  background-size: cover;
  background-repeat: no-repeat;
}
```

## Figma design 
![Screenshot from 2023-06-03 00-11-15](https://github.com/ReyAntonio12/Reynaldo--Portfolio/assets/16392046/b6af0112-6969-4839-a14b-23933742e604)

## Your design 
![Screenshot from 2023-06-03 00-16-42](https://github.com/ReyAntonio12/Reynaldo--Portfolio/assets/16392046/b85b525d-c8ac-4654-81ee-e4a17c58222c)